frontend/src/hooks/useRequestOperations.ts

```
@@ -0,0 +1,109 @@
+    updateError.update(() => null);
+
+    try {
+      await onUpdate(request);
+    } catch (err) {
+      const normalized = err instanceof Error ? err : new Error(String(err));
+      if (resetToken.value === token) {
+        updateError.update(() => normalized);
+      }
+      throw normalized;
+    } finally {
+      if (resetToken.value !== token) return;
```

Biome (2.1.2)

[error] Unsafe usage of 'return'.

'return' in 'finally' overwrites the control flow statements inside 'try' and 'catch'.

(lint/correctness/noUnsafeFinally)